src/container/srv_target.c

```diff
@@ -2201,9 +2201,8 @@ cont_oid_alloc(struct ds_pool_hdl *pool_hdl, crt_rpc_t *rpc)
 	sgl.sg_nr_out = 0;
 	sgl.sg_iovs = &iov;
 
-	rc = oid_iv_reserve(pool_hdl->sph_pool->sp_iv_ns,
-			    in->coai_op.ci_pool_hdl, in->coai_op.ci_uuid,
-			    in->coai_op.ci_hdl, in->num_oids, &sgl);
+	rc = oid_iv_reserve(pool_hdl->sph_pool->sp_iv_ns, in->coai_op.ci_pool_hdl,
+			    in->coai_op.ci_uuid, in->num_oids, &sgl);
```

Comment:
probably use pool uuid, instead of pool handle uuid as well?

Reply:
yea makes sense, let me push an update